src/api/config/environment.rb

```diff
@@ -6,7 +6,14 @@
 path = Rails.root.join('config', 'options.yml')
 
 begin
-  CONFIG = YAML.load_file(path)[Rails.env]
+  config = YAML.load_file(path)
+  if config.key?(Rails.env)
+    CONFIG = config[Rails.env]
+  else
+    # FIXME: Remove with the next stable release (v2.10 or v3.0)
+    Rails.logger.info "DEPRECATED: Please update your options.yml by running 'rake migrate_options_yml'"
```

Comment:
This doesnt work as you don't have a logger in the initializer! You can either define it with:

```
logfile = File.open("#{Rails.root}/log/#{Rails.env}.log", 'a')  
logger = Logger.new(logfile)
```

or leave the puts!